sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -525,11 +532,41 @@ def index_documents(self, batch, **kwargs):
         :type batch: IndexDocumentsBatch
         :rtype:  List[IndexingResult]
         """
-        index_documents = IndexBatch(actions=batch.actions)
+        return self._index_documents_actions(actions=batch.actions, **kwargs)
```

Comment:
Is this a change of behavior for a customer who was using that before? What's the impact if I was user of this?

Reply:
The only difference is when the response is 413 (request is too big). 
old behavior: user will get error
new behavior: we will split the request for user and try again. So it may succeed.